src/main/rx/rx.c

```diff
@@ -403,6 +404,15 @@ STATIC_UNIT_TESTED uint16_t updateLinkQualitySamples(uint16_t value)
     sampleIndex = (sampleIndex + 1) % LINK_QUALITY_SAMPLE_COUNT;
     return sum / LINK_QUALITY_SAMPLE_COUNT;
 }
+
+void rxSetRfMode(uint8_t rfModeValue)
```

Comment:
rfModeValue should be const here:
`void rxSetRfMode(const uint8_t rfModeValue)`

Reply:
This is not necessary.